cockpit/src/cockpit-components-dialog.jsx

```diff
@@ -189,6 +189,7 @@ class DialogFooter extends React.Component {
                 <button
                     className={ cancel_style }
                     onClick={event => this.cancel_click(event)}
+                    disabled={ this.state.action_in_progress?"disabled":"" }
```

Comment:
Why not use variable `action_disabled` here too? Or is there any use case that would not work with this variable?